keps/sig-storage/4049-storage-capacity-scoring-of-nodes-for-dynamic-provisioning/kep.yaml

```diff
@@ -0,0 +1,40 @@
+title: Storage Capacity Scoring of Nodes for Dynamic Provisioning
+kep-number: 4049
+authors:
+  - "@cupnes"
+owning-sig: sig-storage
+participating-sigs:
+status: TBD
+creation-date: TBD
+reviewers:
+  - TBD
+approvers:
+  - TBD
```

Comment:
Can you add reviewers and approvers?

Reply:
Understood. According to [OWNERS file](https://github.com/kubernetes/enhancements/blob/a51e5b8798ac50a98a14bcaa5f7c7d94d9588361/keps/sig-storage/OWNERS), I will add `sig-storage-leads` to `reviewers` and `approvers`.